src/common/StyledCard/index.ts

```diff
@@ -0,0 +1,27 @@
+'use client'
+import styled from '@emotion/styled'
+import { $whiteColor, $blueColor, $pinkColor } from '../../styles/_variables'
+
+export const StyledCard = styled.div`
+  position: relative;
+  padding: 1rem;
+  box-shadow:
```

Comment:
this solution doesnt work.

Reply:
Can you elaborate what you mean by that?